spec/system/user_page/user_preferences_security_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+describe "User preferences for Security", type: :system do
+  fab!(:user) { Fabricate(:user, email: "dude@pm.com", password: "kungfukenny") }
+  let(:user_preferences_security_page) { PageObjects::Pages::UserPreferencesSecurity.new }
+  let(:user_menu) { PageObjects::Components::UserMenu.new }
+
+  before do
+    Fabricate(:email_token, email: "dude@pm.com", user: user, confirmed: true)
```

Comment:
Can we do something like user.primary_email and remove it from fab!(user) too? that way we make the spec more resilient to future changes.

Reply:
Yes, good call, `user.activate` is the way to do this.